src/components/DownloadInvoiceForm.js

```diff
@@ -9,16 +10,31 @@ export default class DownloadInvoiceForm extends Component {
     this.state = {
       start: moment().startOf('day'),
       end: moment().startOf('day'),
-      client: props.clients.first()
+      client: props.clients.first(),
+      address: '',
+      notes: '',
+      userRates: this.userRatesFromProps(props),
```

Comment:
Actually, it's a normalized map of users, but with the `rate` key added. Perhaps `usersWithRates`?